pkgs/applications/misc/raider/default.nix

```diff
@@ -0,0 +1,56 @@
+{ blueprint-compiler
+, desktop-file-utils
+, fetchFromGitHub
+, gettext
+, glib
+, gtk4
+, itstool
+, lib
+, libadwaita
+, meson
+, mesonEmulatorHook
+, ninja
+, pkg-config
+, stdenv
+, wrapGAppsHook4
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "raider";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "ADBeveridge";
+    repo = finalAttrs.pname;
```

Comment:
It probably does not make much sense to couple the repo name to `pname`. If `pname` is changed in an overlay, we would not want to break the source URL.

Reply:
Using `finalAttrs` for version was fine. I just meant using `"raider"` here literally.

Reply:
Yes. I needed finalAttrs for the version test. But `raider --version` does not return a proper version. So i removed it.